analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -414,6 +414,122 @@ public void Invocation_IsNullOrEmpty_TryFinally()
             });
         }
 
+#if NET
+
+        [TestMethod]
+        public void Invocation_TryParse_Null()
+        {
+            const string code = @"
+this.ObjectField = null;
+string byteString = null;
+var success = byte.TryParse(byteString, out var result); // bool TryParse([NotNullWhen(true)] string? s, out byte result)
+Tag(""ByteString"", byteString);
+Tag(""Success"", success);
+Tag(""Result"", result);
+Tag(""ObjectField"", ObjectField);
+";
+            var validator = SETestContext.CreateCS(code).Validator;
+            validator.ValidateTag("ByteString", x => x.HasConstraint(ObjectConstraint.Null).Should().BeTrue());
+            validator.ValidateTag("Success", x => x.HasConstraint(BoolConstraint.False).Should().BeTrue());
+            validator.ValidateTag("Result", x => x.Should().BeNull());
+            validator.ValidateTag("ObjectField", x => x.HasConstraint(ObjectConstraint.Null).Should().BeTrue());
```

Comment:
What's the point of using `ObjectField`? It's practically not used for anything and we can remove it from the test case

Reply:
Same below